### DISCUSSION TITLE
Neolink with RTSP camera?

### DISCUSSION THREAD

**Discussion by 0dragosh** [thirtythreeforty/neolink#276] [Q&A] [answered]
Is there any added benefit to using neolink with a Reolink camera that natively supports RTSP like the RLC-811A?

**Comment by QuantumEntangledAndy** [comment, marked as answer]
Currently not so much. If we get the motion and mqtt merged then yes since you cannot pull motion messages with the official rtsp stream.

Neolink has never explicitly targeted towards those camera though as it was developed to provide rtsp to our non rtsp cameras.

**Reply by 0dragosh** [reply]
Understood, thanks for the quick reply!
